database/seed.js

```diff
@@ -0,0 +1,435 @@
+const db  = require('./index.js');
+const Menu = require('./Restaurants.js');
+// const restaurantsData = require('./restaurantsData.js');
+const faker = require('faker');
+
+
```

Comment:
Looks like the array below is the same data as the `database/restaurantsData.js`?